explorer/src/utils/chainStateParsing.ts

```diff
@@ -16,38 +16,51 @@ export const formatOperators = (
       currentTotalStake: BigInt(op.currentTotalStake).toString(10),
       currentTotalShares: BigInt(op.currentTotalShares).toString(10),
       totalStorageFeeDeposit: BigInt(op.totalStorageFeeDeposit).toString(10),
-      status: JSON.stringify(op.status),
+      status: JSON.stringify(op.status.toString()),
     } as Operators
   })
 
 export const formatDeposits = (deposits: [StorageKey<AnyTuple>, Codec][]) =>
   deposits.map((deposit) => {
+    const depositHeader = deposit[0].toHuman() as [string, string]
     const parsedDeposit = deposit[1].toJSON() as RawDeposit
+    const pending =
+      parsedDeposit.pending !== null
+        ? {
+            effectiveDomainId: parsedDeposit.pending.effectiveDomainEpoch[0],
+            effectiveDomainEpoch: parsedDeposit.pending.effectiveDomainEpoch[1],
+            amount: BigInt(parsedDeposit.pending.amount),
+            storageFeeDeposit: BigInt(parsedDeposit.pending.storageFeeDeposit),
+          }
+        : null
     return {
-      operatorId: parseInt((deposit[0].toHuman() as string[])[0]),
-      account: (deposit[0].toHuman() as string[])[1],
-      shares: BigInt(parsedDeposit.known.shares).toString(10),
-      storageFeeDeposit: parseInt(parsedDeposit.known.storageFeeDeposit.toString(), 16).toString(),
-      pending: {
-        amount: BigInt(parsedDeposit.pending.amount).toString(10),
-        storageFeeDeposit: BigInt(parsedDeposit.pending.storageFeeDeposit).toString(10),
+      operatorId: parseInt(depositHeader[0]),
+      account: depositHeader[1],
+      shares: BigInt(parsedDeposit.known.shares.toString()),
+      storageFeeDeposit: BigInt(parsedDeposit.known.storageFeeDeposit.toString()),
+      known: {
+        shares: BigInt(0), // BigInt(parsedDeposit.known.shares.toString()),
+        storageFeeDeposit: BigInt(0), // BigInt(parsedDeposit.known.storageFeeDeposit.toString()),
```

Comment:
Are these intended to be `BigInt(0)`?

Reply:
oups I forgot I had commented these lines

after the first toJson() they are number, but since all our math is in bitint, I'm converting them